test/src/e2e.dynval/dv.n'.test.ts

```diff
@@ -17,39 +17,26 @@
 import * as chai from "chai";
 import { expect } from "chai";
 import * as chaiAsPromised from "chai-as-promised";
-import { blake256, H256 } from "codechain-primitives/lib";
-import { SDK } from "codechain-sdk";
 import * as stake from "codechain-stakeholder-sdk";
 import "mocha";
 
 import { validators as originalDynValidators } from "../../tendermint.dynval/constants";
 import {
-    aliceSecret,
-    bobSecret,
     faucetAddress,
     faucetSecret,
-    stakeActionHandlerId,
-    validator0Address,
-    validator1Address,
-    validator2Address,
-    validator3Address
 } from "../helper/constants";
-import { PromiseExpect, wait } from "../helper/promise";
-import CodeChain from "../helper/spawn";
+import { PromiseExpect } from "../helper/promise";
 import { withNodes } from "./setup";
 
 chai.use(chaiAsPromised);
 
-const RLP = require("rlp");
-
 const [
     alice,
     betty,
-    notUsed,
-    notUsed2,
+    ,
+    ,
```

Comment:
I think some patches in this commit (including this line) should be moved to previous commits.
Please move them if you can.